src/pages/docs/Code-of-conduct.mdx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import Link from 'next/link'
+ 
+# Code of Conduct
```

Comment:
is it possible to import the existing CODE_OF_CONDUCT.md file into this file so that we have one source of truth for this documentation?

i say this because if we want to update something in our code of conduct process, we now will have two files to update the same information